x-pack/plugins/monitoring/server/kibana_monitoring/bulk_uploader.js

```diff
@@ -151,7 +151,7 @@ export class BulkUploader {
 
     const data = await usageCollection.bulkFetch(this._cluster.callAsInternalUser);
     const payload = this.toBulkUploadFormat(compact(data), usageCollection);
-    if (payload) {
+    if (payload.length > 0) {
```

Comment:
```suggestion
    if (payload && payload.length > 0) {
```
Just to be safe in the off chance that something else returns fulsey in the formatter